### PR TITLE
slight modification to go.* and README

### PR DESCRIPTION
**Pull request by kris-watts-gravwell**
showing how to inject version on docker build.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -12,7 +12,7 @@ To build just the PDF:
 
 To build the whole tarball with docker images and all:
 
-	LICENSE=/path/to/gravwell-license make dist
+	VERSION=5.2.2 LICENSE=/path/to/gravwell-license make dist
```
Comment (ashnwade) [inline README:15]: should this be 5.3.0 for the new release?

**Review by ashnwade — COMMENTED** [review]
Question

File: README (hunk 1 above)
Comment (kris-watts-gravwell) [reply README]: yep

**Review by ashnwade — APPROVED** [review]
LGTM

**Pull request merged**